tests/lib/eslint.js

```diff
@@ -2193,6 +2193,11 @@ describe("eslint", function() {
             assert.equal(messages.length, 0);
         });
 
+        it("should send sourceType \"module\" to escope when there is an ecmaFeatures.modules (plural) and ecmaFeatures.blockBindings attributes in the config", function() {
```

Comment:
should properly parse import statements when ecmaFeatures.modules AND 
blockBindings are true

On 07/03/2015 21:08, Nicholas C. Zakas wrote:
> should properly parse import statements when ecmaFeatures.modules is true